main1.js

```diff
@@ -0,0 +1,38 @@
+// 1a -- nieuwe array maken met tv type
+console.log('OPDRACHT 1a');
+
+const tvName = inventory.map((item) => {
+    return item.name;
+});
+
+console.log(tvName);
+
+// 1b -- nieuwe array maken met uitverkochte tv's
+console.log('OPDRACHT 1b');
+
+const tvOutOfStock = inventory.filter((item) => {
+    if (item.originalStock - item.sold === 0) {
+        return item;
+    }
+});
```

Comment:
De .filter functie verwacht een boolean als resultaat https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#parameters
```suggestion
const tvOutOfStock = inventory.filter((item) => item.originalStock - item.sold === 0);
```